src/Manager/Environment/effects/removeDir.ts

```diff
@@ -6,25 +6,19 @@
 
 import fse from 'fs-extra';
 import path from 'path';
-import { ErrorDialogActions, usageData } from 'pc-nrfconnect-shared';
 
-import { Dispatch } from '../../../state';
-
-export const removeDir = async (dispatch: Dispatch, srcDir: string) => {
-    let renameOfDirSuccessful = false;
+export const removeDir = (srcDir: string) => {
+    const toBeDeletedDir = path.resolve(srcDir, '..', 'toBeDeleted');
     try {
-        const toBeDeletedDir = path.resolve(srcDir, '..', 'toBeDeleted');
-        await fse.move(srcDir, toBeDeletedDir, { overwrite: true });
-        renameOfDirSuccessful = true;
-        await fse.remove(toBeDeletedDir);
+        fse.renameSync(srcDir, toBeDeletedDir);
+        fse.removeSync(toBeDeletedDir);
```

Comment:
Recently the issue came up, that nowadays `fs-extra` gives only little benefit over plain `node:fs`. So, if you are anyway working on code that uses `fs-extra`, feel free to also switch that code to use just `node:fs`.

E.g. here you could replace the import of `fs-extra` to `fs` and then do this:

```suggestion
        fs.renameSync(srcDir, toBeDeletedDir);
        fs.rmSync(toBeDeletedDir, { recursive: true, force: true });
```